fast64_internal/f3d/f3d_writer.py

```diff
@@ -874,10 +875,10 @@ def __init__(
         texDimensions,
         material: bpy.types.Material,
         currentGroupIndex,
-        triList,
-        vtxList,
+        triGroup,
         existingVertexData,
         existingVertexMaterialRegions,
+        celShadingInfo
```

Comment:
if you create a `CelShadingInfo` class this would be a great place for a type annotation